lib/resources/example_json/dstu2_examples_allergy_intolerance.rb

```diff
@@ -1,300 +1,219 @@
 module Cerner
   module Resources
 
-    DSTU2_ALLERGY_INTOLERANCE_BUNDLE ||= {
-      "resourceType": "Bundle",
-      "id": "c7330db8-59df-499b-9f97-9f89b79dbfb4",
-      "type": "searchset",
-      "total": 2,
-      "link": [
-        {
-          "relation": "self",
-          "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance?patient=4342010"
-        }
-      ],
-      "entry": [
-        {
-          "fullUrl": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/5703737",
-          "resource": {
-            "resourceType": "AllergyIntolerance",
-            "id": "5703737",
-            "meta": {
-              "versionId": "5703737",
-              "lastUpdated": "2016-06-27T14:10:11.000Z"
-            },
-            "text": {
-              "status": "generated",
-              "div": "&lt;div>&lt;p>&lt;b>Allergy Intolerance&lt;/b>&lt;/p>&lt;p>&lt;b>Patient&lt;/b>: Smart, Joe&lt;/p>&lt;p>&lt;b>Allergy&lt;/b>: Peanuts&lt;/p>&lt;p>&lt;b>Status&lt;/b>: Active&lt;/p>&lt;p>&lt;b>Criticality&lt;/b>: High Risk&lt;/p>&lt;p>&lt;b>Category&lt;/b>: Food&lt;/p>&lt;p>&lt;b>Reactions&lt;/b>: Dyspnea&lt;/p>&lt;p>&lt;b>Onset&lt;/b>: 1982&lt;/p>&lt;/div>"
-            },
-            "onset": "1982",
-            "recordedDate": "2016-06-27T09:10:11.000-05:00",
-            "recorder": {
-              "reference": "Practitioner/1912007",
-              "display": "Who, Doctor"
-            },
-            "patient": {
-              "reference": "Patient/4342010",
-              "display": "Smart, Joe"
-            },
-            "substance": {
-              "coding": [
+    DSTU2_ALLERGY_INTOLERANCE_RESOURCE ||= {
+        "resourceType": "AllergyIntolerance",
+        "id": "12760025",
+        "meta": {
+            "versionId": "12760025",
+            "lastUpdated": "2020-03-04T20:16:02.000Z"
+        },
+        "text": {
+            "status": "generated",
+            "div": "<div><p><b>Allergy Intolerance</b></p><p><b>Patient</b>: SMART, SANDY A</p><p><b>Allergy</b>: Egg</p><p><b>Status</b>: Active</p><p><b>Criticality</b>: High Risk</p><p><b>Category</b>: Food</p><p><b>Reactions</b>: Breathing abnormal</p><p><b>Onset</b>: Dec 2019</p></div>"
```

Comment:
All of these divs need to be URL encoded in every example. Should just be a couple find & replace-all's in the dir

Reply:
See https://github.com/cerner/fhir.cerner.com/pull/420#issuecomment-655776349

Reply:
Also it's not URL encoded but HTML encoding. https://stackoverflow.com/questions/1812473/difference-between-url-encode-and-html-encode

Reply:
See https://github.com/cerner/fhir.cerner.com/pull/420/files#diff-7718c11ed130ac00a9854e0b089f74b4R90